dev/export-db.sh

```
@@ -0,0 +1,111 @@
+# Load local .env or use defaults
+if [ -f .env ]; then
+    set -a
+    source .env
+    set +a
+fi
+
+LOCAL_HOST="${METRICS_DB_HOST:-localhost}"
+LOCAL_PORT="${METRICS_DB_PORT:-15432}"
+LOCAL_DB="${METRICS_DB_NAME:-github_metrics_dev}"
+LOCAL_USER="${METRICS_DB_USER:-postgres}"
+LOCAL_PASSWORD="${METRICS_DB_PASSWORD:-dev}"
```

Shellcheck (0.11.0)

[warning] LOCAL_PASSWORD appears unused. Verify use (or export if used externally).

(SC2034)